### ISSUE TITLE
Account Details don't save when posting a trade

### ISSUE THREAD

**Issue by tobias-g1** [nirvanaitsolutions/swapsteem#63]
**Describe the bug**
When posting a trade, bank account details don't save. 

**To Reproduce**
Steps to reproduce the behaviour:
1. Post a trade
2. Select the type of Buy
3. Fill in all empty values with the value "1"
4. Publish the trade. 
5. Once created, go back and view/edit your recently created trade. 

**Expected behaviour**
All details entered the form should be saved and ready

**Screenshots**

![jan-26-2019 12-00-09](https://user-images.githubusercontent.com/17148879/51786852-3454e000-2162-11e9-9a0e-36fb2442085c.gif)

**Desktop (please complete the following information):**
Browser: Google Chrome (Version 71.0.3578.98 (Official Build) (64-bit))
Device: MacBook Pro (Retina, 15-inch, Late 2013)
Operating system: MacOS Mojave Version 10.14.1 (18B75)

**Additional context**
Looks like an unimplemented feature, as in the request it doesn't seem to be sent. 

```
{"createdby":"votezilla","ad_type":"BUY","country":"","payment_methods":"Bank Transfer","currency":"USD","margin":0,"limit_from":0,"limit_to":0,"restricted_amounts":[],"ad_coin":"STEEM","ad_status":"open","ad_coin_amount":1,"terms":"1","ad_details":{"minimum_volume":0,"minimum_reputation_score":25,"new_buyer_limit":0,"track_liquidity":true},"security_details":{"identified_people_only":true,"identify_user_before_continuing_trade":true,"real_name_required":true,"sms_verification_required":true,"trusted_people_only":true}}
~``

**Comment by Shubh1692** [comment]
Hello, @tobias-g1 this functionality has implemented and deployed n issue #54.

**Comment by nirvanaitsolutions** [comment]
Cool @Shubh1692. Closing this issue.
